extensions/3rdparty/ArrayExtension/ArrayExtension.php

```diff
@@ -380,7 +380,7 @@ function arrayreset( &$parser, $arrayids) {
 *
 *   see: http://www.php.net/manual/en/function.array-unique.php
 */
-    function arrayunique( &$parser, $arrayid ) {
+    function arrayunique( $parser, $arrayid ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$parser". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)